c/tests/test.cc

```diff
@@ -33,12 +33,17 @@ TEST(BasicTest, BasicTest) {
   EXPECT_EQ(strlen(reinterpret_cast<const char *>(data_ptr)), data_size);
   EXPECT_EQ(
       reinterpret_cast<const char *>(data_ptr),
-      std::string("Error at plan: failed to parse as substrait.Plan: "
-                  "failed to decode Protobuf message: "
-                  "invalid wire type value: 7 (code 1001) (code 1001)\n"
-                  "Error at plan: failed to parse as substrait.PlanVersion: "
-                  "failed to decode Protobuf message: "
-                  "invalid wire type value: 7 (code 1001) (code 1001)\n"));
+      std::string(
+          "Info at plan: this version of the validator is EXPERIMENTAL. "
```

Comment:
Note that before the `EXPERIMENTAL` info was not printed when the substrait plan failed to decode into either a `Plan` or `PlanVersion`. Given that it is otherwise always printed, it seemed more consistent to have it always be printed (even under full decode failures) and update this test, rathen than to make the code match this test.